datasets/multi_booked/multi_booked.py

```diff
@@ -0,0 +1,164 @@
+    booktitle = {Proceedings of the Eleventh International Conference on Language Resources and Evaluation (LREC'18)},
+    year = {2018},
+    month = {May},
+    date = {7-12},
+    address = {Miyazaki, Japan},
+    publisher = {European Language Resources Association (ELRA)},
+    language = {english}
+}
+"""
+
+_DESCRIPTION = """\
+MultiBooked is a corpus of Basque and Catalan Hotel Reviews Annotated for Aspect-level Sentiment Classification.
+
+The corpora are compiled from hotel reviews taken mainly from booking.com. The corpora are in Kaf/Naf format, which is
+an xml-style stand-off format that allows for multiple layers of annotation. Each review was sentence- and
+word-tokenized and lemmatized using Freeling for Catalan and ixa-pipes for Basque. Finally, for each language two
+annotators annotated opinion holders, opinion targets, and opinion expressions for each review, following the
+guidelines set out in the OpeNER project.
+"""
+
+_HOMEPAGE = "http://hdl.handle.net/10230/33928"
+
+_LICENSE = "CC-BY 3.0"
+
+_URL = "https://github.com/jerbarnes/multibooked/archive/master.zip"
+
+
+class MultiBooked(datasets.GeneratorBasedBuilder):
+    """MultiBooked dataset."""
+
+    VERSION = datasets.Version("0.0.0")
+
+    BUILDER_CONFIGS = [
+        datasets.BuilderConfig(name="ca", description="MultiBooked dataset in Catalan language."),
+        datasets.BuilderConfig(name="eu", description="MultiBooked dataset in Basque language."),
+    ]
+
+    def _info(self):
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=datasets.Features(
+                {
+                    "text": datasets.features.Sequence(
+                        {
+                            "wid": datasets.Value("string"),
+                            "sent": datasets.Value("string"),
+                            "para": datasets.Value("string"),
+                            "word": datasets.Value("string"),
+                        }
+                    ),
+                    "terms": datasets.features.Sequence(
+                        {
+                            "tid": datasets.Value("string"),
+                            "lemma": datasets.Value("string"),
+                            "morphofeat": datasets.Value("string"),
+                            "pos": datasets.Value("string"),
+                            "target": datasets.features.Sequence(datasets.Value("string")),
+                        }
+                    ),
+                    "opinions": datasets.features.Sequence(
+                        {
+                            "oid": datasets.Value("string"),
+                            "opinion_holder_target": datasets.features.Sequence(datasets.Value("string")),
+                            "opinion_target_target": datasets.features.Sequence(datasets.Value("string")),
+                            "opinion_expression_polarity": datasets.Value("string"),
```

Comment:
Could you use the ClassLabel feature type here ?
For example
```
"opinion_expression_polarity": datasets.ClassLabel(names=["Negative", "Positive"])
```

you'll need to regenerate the dataset_infos.json file after this change

Also note that if there are missing values, you should use the integer `-1` (currently the script returns an empty string for missing polarities)

Reply:
@lhoestq also done. There aren't missing values.